app/src/androidTest/java/HomePage.kt

```diff
@@ -1,8 +1,6 @@
 import com.bijoysingh.quicknote.R
 
-open class HomePage : EspressoBaseTest() {
+open class HomePage {
 
-    fun getRecyclerView(): Int {
-        return R.id.recycler_view
-    }
+    open val getRecyclerView = R.id.recycler_view
```

Comment:
The name should be `recyclerView`, because it isn't getter method, but just val. 
It shouldn't be `open`, because `HomePage` isn't overriden in child class, because child class just doesn't exist :). `HomePage` isn't parent for any class.